core/acl.py

```diff
@@ -113,3 +119,34 @@ async def repeat(self, ctx, *, input: str):
 
     # user's roles are not mapped to any ACL group, return default
     return rule.default
+
+
+def version(major: int, minor: int, micro: Optional[int] = None):
+    """Decorator that specifies minimum discord.py version"""
+
+    def decorator(func):
+        @wraps(func)
+        async def wrapper(*args, **kwargs):
+            version_info = discord.version_info
+            version_check = True
+
+            if version_info.major == major:
+                if version_info.minor == minor:
+                    if micro is None or version_info.micro >= micro:
```

Comment:
Shouldn't this be `if micro is not None and version_info.micro >= micro`?

Reply:
My bad, was thinking about checking for wrong version at first and then I did this version.